app/src/main/java/org/oppia/android/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -45,17 +46,19 @@ class SelectionInteractionViewModel private constructor(
   private val minAllowableSelectionCount: Int by lazy {
     interaction.customizationArgsMap["minAllowableSelectionCount"]?.signedInt ?: 1
   }
-  private val maxAllowableSelectionCount: Int by lazy {
+  val maxAllowableSelectionCount: Int by lazy {
     // Assume that at least 1 answer always needs to be submitted, and that the max can't be less than the min for cases
     // when either of the counts are not specified.
     interaction.customizationArgsMap["maxAllowableSelectionCount"]?.signedInt
       ?: minAllowableSelectionCount
   }
-  private val selectedItems: MutableList<Int> = mutableListOf()
+  val selectedItems: MutableList<Int> = mutableListOf()
   val choiceItems: ObservableList<SelectionInteractionContentViewModel> =
     computeChoiceItems(choiceSubtitledHtmls, hasConversationView, this)
 
   private val isAnswerAvailable = ObservableField(false)
+  val selectedItemText = ObservableField("Please select all correct choices")
+  val enabledItemsList by lazy { List(choiceItems.size) { ObservableBoolean() } }
```

Comment:
Shouldn't these be default enabled, e.g. by initializing with ``ObservableBoolean(true)``?

Reply:
Okay Ben

Reply:
I think these are false at the moment (ObservableBoolean is false by default)--is that correct?

Reply:
done.